api/recompress_chunk.md

```diff
@@ -25,6 +25,12 @@ You can also recompress chunks by
 `recompress_chunk` gives you more fine-grained control by
 allowing you to target a specific chunk.
 
+<Highlight type="important">
```

Comment:
Add a link to `recompress_chunk` until it is removed?

Reply:
Sorry, no, I just noted that a link was missing. Didn't notice that it was to the same page.